packages/camera/camera_avfoundation/test/avfoundation_camera_test.dart

```diff
@@ -361,77 +343,27 @@ void main() {
       final StreamQueue<CameraInitializedEvent> streamQueue =
           StreamQueue<CameraInitializedEvent>(eventStream);
 
+      final PlatformSize previewSize = PlatformSize(width: 3840, height: 2160);
       // Emit test events
       final CameraInitializedEvent event = CameraInitializedEvent(
         cameraId,
-        3840,
-        2160,
+        previewSize.width,
+        previewSize.height,
         ExposureMode.auto,
         true,
         FocusMode.auto,
         true,
       );
-      await camera.handleCameraMethodCall(
-          MethodCall('initialized', event.toJson()), cameraId);
-
-      // Assert
-      expect(await streamQueue.next, event);
-
-      // Clean up
-      await streamQueue.cancel();
-    });
-
-    test('Should receive resolution changes', () async {
```

Comment:
This and the next test are testing functionality that was dead code. The Dart method call handler handled several messages that are never sent by the native code. It doesn't look like Android uses them either, so I'm guessing it was code mistakenly left in during refactoring in code review of a feature.